app/views/full-page-examples/what-is-your-nationality/index.js

```diff
@@ -23,7 +23,22 @@ module.exports = (app) => {
     [
       body('confirm-nationality')
         .exists()
-        .not().isEmpty().withMessage('Select your nationality or nationalities')
+        .not().isEmpty().withMessage('Select your nationality or nationalities'),
+      body('country-name')
```

Comment:
This could mirror the logic in `feedback-page`

```javascript
body('country-name')
    .custom((value, { req: request }) => {
        // See https://github.com/express-validator/express-validator/pull/658
        const otherCountryNationality = request.body['confirm-nationality']
        if (!otherCountryNationality.includes('other-country-nationality')) {
        return true
        }
        if (!value) {
        throw new Error('Enter your country')
        }
        return true
    })
```

Reply:
This logic is not right one second...

Reply:
Potentially this is slightly easier to read since it returns early rather than having a forking conditional.

Up to you though, I don't think this is blocking.

Reply:
Nice 👍 Have refactored accordingly.